packages/ubuntu_desktop_installer/lib/pages/installation_complete/installation_complete_page.dart

```diff
@@ -68,6 +71,8 @@ class InstallationCompletePage extends StatelessWidget {
                 ),
                 OutlinedButton(
                   onPressed: () {
+                    Provider.of<TelemetryService>(context, listen: false)
+                        .done();
                     Provider.of<InstallationCompleteModel>(context,
                             listen: false)
                         .shutdown(immediate: false);
```

Comment:
Not against it, but `InstallationCompleteModel` doesn't have a `done()` method yet. Since there are two possible actions (shutdown and reboot), such a method would need an enum parameter. Not sure we're gaining anything here in terms in code readability/maintenance. Or am I missing your point?

Reply:
I'm usually trying to hide service access from the UI to the view models but I didn't think about shutdown vs. reboot. Feel free to ignore this. :)